bats/fb-test-puppet.bats

```diff
@@ -61,7 +61,7 @@ fi
 }
 
 @test "import ntp puppet class" {
-  id=$(hammer --csv proxy list | tail -n1 | cut -d, -f1)
+  id=$(hammer --csv proxy list --search $(hostname -f) | tail -n1 | cut -d, -f1)
```

Comment:
I'd expect you should be able to use `hamer proxy info --name $(hostname -f)` and not need to use tail. You might even be able to simplify it all to `hammer proxy import-classes --name $(hostname -f)`.

Reply:
Totally, right that works great.